packages/kbn-test/src/mocha/junit_report_generation.js

```diff
@@ -140,11 +140,7 @@ export function setupJUnitReportGeneration(runner, options = {}) {
       };
 
       // adding code owners only for the failed test case
-      if (failed) {
-        const testCaseRelativePath = getPath(node);
-        const owners = getCodeOwnersForFile(testCaseRelativePath, reversedCodeowners);
-        attrs.owners = owners || ''; // empty string when no codeowners are defined
-      }
+      if (failed) attrs.owners = getCodeOwnersForFile(getPath(node), reversedCodeowners);
```

Comment:
I think we shouldn't add the object here as `attrs.owners` as we're not interested in the matching path here.